test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts

```diff
@@ -0,0 +1,57 @@
+describe('In Defense of Kamino', function () {
+    integration(function (contextRef) {
+        describe('In Defense of Kamino\'s ability', function () {
+            it('should give each friendly Republic unit Restore 2 and: "When Defeated: Create a Clone Trooper token."', function () {
```

Comment:
nit: can merge the `describe` into the `it`

Reply:
done